spec/value_objects/value_object_spec.rb

```diff
@@ -77,15 +73,21 @@
   end
 
   describe 'inquiry methods' do
-    let(:fruit_one) { Fruit.new(:apple) }
-    let(:fruit_two) { Fruit.new(:banana) }
+    let(:yes_answer) { YesNoAnswer.new(:yes) }
+    let(:no_answer)  { YesNoAnswer.new(:no) }
+
+    it 'has a constant with the inquiry methods' do
+      expect(
+        YesNoAnswer::INQUIRY_METHODS
```

Comment:
This is interesting to me - I like it but I'm struggling to understand the application other than in this test 🤔

Reply:
I use it here (introduced in this PR):
https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/pull/270/files#diff-88c2fd3a91d8fa60265ec5797a508c95bc1d6a4eea137880bc79ccf37f7cf0f1R3

😅